### DISCUSSION TITLE
Block URLs found on FI's Investor alerts list

### DISCUSSION THREAD

**Discussion by ghost** [lassekongo83/Frellwits-filter-lists#149] [Ideas]
Hi,

With investment fraud attempts being and becoming even more common, I thought it could be a good idea for Frellwit's Swedish Filter to include all URLs found on FI's (Finansinspektionen) Investor alerts list: https://www.fi.se/en/our-registers/investor-alerts/. Note that not all entries on FI's list include a URL, and that the list is updated relatively often.

Thanks for providing and maintaining the Swedish filter!

Best regards,
Mårten

**Comment by lassekongo83** [comment]
Om listan på URLs fanns som ren textfil nånstans så skulle det nog gå.
